include/cantera/oneD/StFlow.h

```diff
@@ -242,6 +242,12 @@ class StFlow : public Domain1D
         m_kin->getNetProductionRates(&m_wdot(0,j));
     }
 
+    virtual void updateProperties(size_t jg, double* x, double* rsd, int* diag,
+                                  double rdt, size_t jmin, size_t jmax);
```

Comment:
I think you can simplify this function signature. Updating the thermo/transport properties shouldn't require `rsd`, `diag`, or `rdt`, right? Also, this function needs a docstring.